.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
         files: \.(jsx?|tsx?|css|.md)$
         exclude: \.*__generated__.*$
         additional_dependencies:
-          - prettier@3.2.4
+          - prettier@3.3.3
```

Comment:
For some reason prettier in pre-commit had a different stylistic opinion than prettier in my editor until I bumped this version to match our package.json

I am surprised that they seemingly made stylistic changes in a minor version?

Reply:
i keep forgettting we have pre-commit hooks, how do you set  them up again?

Reply:
`pre-commit install` you have to install pre-commit globally somehow first. I get mine from my activated python virtual environment but I believe you can also install it via brew